app/templates/crate/versions.hbs

```diff
@@ -23,3 +23,15 @@
     </li>
   {{/each}}
 </ul>
+{{#if this.next_page}}
+  <div local-class="load-more">
+    <button type="button" data-test-id="load-more" disabled={{this.loadMoreTask.isRunning}}
+      {{on "click" (perform this.loadMoreTask)}}
+    >
+      Load More
+      {{#if this.loadMoreTask.isRunning}}
+        <LoadingSpinner />
+      {{/if}}
```

Comment:
it looks like the spinner isn't vertically centered correctly. it looks a bit off.

Reply:
Thanks, I'll take a look!